packages/dscc-scripts/.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
```

Comment:
Hmm, I wonder how Travis works with Monorepos, we might have to figure out a better way to trigger test-runs.

Reply:
Also, it's fine for travis to be broken for now. We can try to just fix it soon, I wouldn't mix in fixing travis with these commits.